syncd/Syncd.cpp

```diff
@@ -4960,6 +4995,22 @@ void Syncd::run()
                     continue;
                 }
 
+                if (shutdownType == SYNCD_RESTART_TYPE_PRE_EXPRESS_SHUTDOWN)
+                {
+                    SWSS_LOG_NOTICE("express boot, enable fast API pre-shutdown");
+                    status = setFastAPIEnableOnAllSwitches();
```

Comment:
why there is API name involved here ? should this be setExpressShutdownOnAllSwitches etc ? or this express shutdown somehow is related to api execution ? we need explanation here

Reply:
Because the function actually sets SAI_SWITCH_ATTR_FAST_API_ENABLE  to true. This is used by platform to prepare for express shutdown.